### PR TITLE
Build env update (rempi) plus new py-crsim package

### PR DESCRIPTION
**Pull request by MakotoSeino**
install for Fugaku.

### CODE REVIEW

**Comment by iarspider** [comment]
@MakotoSeino the `package.py` file is not in correct directory - it should not be in repo root

**Comment by MakotoSeino** [comment]
Change the placement of the two package.py.

File: package.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
```
Comment (iarspider) [inline package.py:6]: Please remove this boilerplate (lines 6-21)

File: var/spack/repos/builtin/packages/py-crsim/package.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
```
Comment (iarspider) [inline var/spack/repos/builtin/packages/py-crsim/package.py]: Same here - remove lines 6-21

File: package.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline package.py:1]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/package.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/package.py:1]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/py-crsim/package.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-crsim/package.py:1]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

**Review by tldahlgren — COMMENTED** [review]
Only addressing the dated copyright notice.

Will defer final approval to @adamjstewart and or @iarspider

File: var/spack/repos/builtin/packages/package.py (hunk 4 above)
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/package.py:1]: You need to remove this package from this directory in your branch.

File: package.py (hunk 3 above)
Comment (tldahlgren) [inline package.py:1]: You need to remove this file from this directory in your branch.

**Review by tldahlgren — CHANGES_REQUESTED** [review]

File: package.py (hunk 3 above)
Comment (tldahlgren) [inline package.py:1]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment (tldahlgren) [inline package.py:1]: This file needs to be removed.

**Comment by adamjstewart** [comment]
@MakotoSeino is this PR still a work in progress?

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**